torch/nn/modules/activation.py

```diff
@@ -1113,7 +1113,7 @@ def forward(self, query: Tensor, key: Tensor, value: Tensor, key_padding_mask: O
                 why_not_fast_path = "some Tensor argument has_torch_function"
             elif not all([(x is None or x.is_cuda or 'cpu' in str(x.device)) for x in tensor_args]):
```

Comment:
I think we need the list comprehension because of torchscriptability, as generators don't work in TS and we have a substantial number of users still depending on TS. :(

Reply:
ooof, makes sense.

Reply:
:(